rm/ut/rmut.c

```diff
@@ -274,16 +279,13 @@ void rm_ctx_server_windup(enum rm_server srv_id)
 {
 	struct m0_rm_owner *owner = rm_ctxs[srv_id].rc_test_data.rd_owner;
 	enum rm_server      cred_id = rm_ctxs[srv_id].creditor_id;
-	struct m0_reqh     *reqh = &rm_ctxs[srv_id].rc_rmach_ctx.rmc_reqh;
 
-	m0_chan_lock(&reqh->rh_conf_cache_exp);
```

Comment:
Why is this needed in the original code? and what changes makes it not necessary now?

Reply:
We now take the lock directly at m0_rm_remote_fini(), if needed.